mkdocs.yml

```diff
@@ -230,11 +230,16 @@ plugins:
   # - section-index
 
 extra_javascript:
-  - https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js
-  - javascripts/tablesort.js
+  - https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.0/jquery.min.js
+  - https://cdn.datatables.net/1.13.4/js/jquery.dataTables.min.js  
```

Comment:
do you still need the datatables jquery scripts?

Reply:
I think you deleted the tablesort javascripts here on accident, which removes the ability to sort columns.  

Could you re-add
```  
  - https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js
  - javascripts/tablesort.js
```

then, could you check if you need these two lines:

```
  - https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.0/jquery.min.js
  - https://cdn.datatables.net/1.13.4/js/jquery.dataTables.min.js  
```

My guess is that you may not need the `- https://cdn.datatables.net/1.13.4/js/jquery.dataTables.min.js`

Thanks!